.github/workflows/scanning.yml

```diff
@@ -0,0 +1,22 @@
+name: Vulnerability scanning
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+
+jobs:
+  build:
```

Comment:
nitpick: `build` is not the right job name

Reply:
Fixed.